### DISCUSSION TITLE
Add Entity Hider option for Others' Quest NPCs

### DISCUSSION THREAD

**Discussion by fuzxi** [runelite/runelite#17115] [Ideas]
Similar to random events, these NPCs pop up for a specific player and can cover up the clickboxes for nearby objects. Non-exhaustive list:
[Desert Treasure 1 Stranger](https://oldschool.runescape.wiki/w/Stranger)
[Desert Treasure 2 Mysterious Figure](https://oldschool.runescape.wiki/w/Mysterious_Figure)
[Regicide King's messenger](https://oldschool.runescape.wiki/w/King%27s_Messenger)
[The Frozen Door Messenger](https://oldschool.runescape.wiki/w/Messenger_(The_Frozen_Door))
[Into the Tombs Messenger](https://oldschool.runescape.wiki/w/Messenger_(Into_the_Tombs)) (this one spams overhead dialogue as well and spawns very frequently if not spoken to - roughly every 3 minutes).

**Comment by YvesW** [comment]
FYI, random-event-hider has an option to hide others' messengers.

**Reply by fuzxi** [reply]
Nice, thank you for the tip!
